eval/eval.go

```diff
@@ -1426,3 +1427,11 @@ func (s *State) stopOutputBuffering() []byte {
 	s.env.PrevOut = nil
 	return output
 }
+
+func isAssignment(node ast.InfixExpression) bool {
```

Comment:
well if you make the signatute the token type instead that would avoid the a bit jarring 
```
if isAssignment(*node) {
```
about as well as save a bunch .Token.Type() here